SplunkRumWorkspace/SplunkRum/SplunkRum/NetworkInstrumentation.swift

```diff
@@ -199,7 +199,7 @@ extension URLSession {
         return answer
        }
 
-    @objc(splunkSwizzledDownloadTaskWithRequest: completionHandler:) open func splunk_swizzled_downloadTask(with request: URLRequest, completionHandler: @escaping (URL?, URLResponse?, Error?) -> Void) -> URLSessionDownloadTask {
```

Comment:
PR Contains two points.
1. flag to enable/disable Network instrumentation.
2. The second is application crashes in iOS 12 because the networking library based on URLSession does not report HTTP requests (Alamofire SDK).

Reply:
I will create a separate PR for completion handle issues.
yes, I will make the upload task optional.